lib/controllers/cats.js

```diff
@@ -1,3 +1,33 @@
 const { Router } = require('express');
+const { cats } = require('../cats-data');
+
+module.exports = Router()
+  .get('/:id', (req, res) => {
+    let match;
+    for (const cat of cats) {
+      if (cat.id === req.params.id) {
+        match = cat;
+      }
+    }
```

Comment:
Using a `for` loop here is fine, but don't forget you have all the array methods at your disposal. For this section you could have used the `.find()` method. 

```js 
const match = cats.find(cat => cat.id === req.params.id);